From ef2efe2883cb899d1bcf8d2f68905304d1dc74e4 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Fri, 23 Sep 2011 10:51:53 +0800
Subject: ENGR00157467: i.MX6/defconfig: enable ARM errata:754322

We need enable the following ARM errata software workaround:
754322: Possible faulty MMU translations following an ASID switch.

CONFIG_ARM_ERRATA_754322=y

These ERRATAs applied to i.MX6Q(cortex-a9:r2p10 smp)

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index d40a01e..49f9218 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Mon Sep 19 16:32:11 2011
+# Fri Sep 23 10:51:09 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -357,6 +357,7 @@ CONFIG_ARM_ERRATA_743622=y
 CONFIG_ARM_ERRATA_751472=y
 # CONFIG_ARM_ERRATA_753970 is not set
 # CONFIG_PL310_ERRATA_727915 is not set
+CONFIG_ARM_ERRATA_754322=y
 CONFIG_ARM_GIC=y
 
 #
-- 
1.7.9.5

